sources/Launcher/DYNRobustnessAnalysisLauncher.cpp

```diff
@@ -92,17 +97,24 @@ RobustnessAnalysisLauncher::init() {
     throw DYNAlgorithmsError(DirectoryDoesNotExist, workingDirectory_);
   workingDirectory_ += '/';  // to be sure to have an '/' at the end of the path
 
+  // build the name of the outputFile
+  outputFileFullPath_ = createAbsolutePath(outputFile_, workingDirectory_);
+
+  // If we already have received a definition for multipleJobs we have finished
+  if (multipleJobs_) {
```

Comment:
Here we assume that if the input is a zip the extraction was already done.
Maybe we should put the extraction code (line 120 to 133) into a separate method so that external users do not have to replicate it when building the multipleJobs instance?

Reply:
In fact what we have done is extend the class `RobustnessAnalysisLauncher` to accept EITHER an object multiple jobs definition or an input file containing it. Previously, the `RobustnessAnalysisLauncher` only accepted an input file name, that could be the xml for the multiple jobs definition or a zip file containing it, that should be named `fic_MULTIPLE.xml`. 

We  have separated the code that unzips the input in a specific method. It also makes more clear the logic in the `RobustnessAnalysisLauncher::init` method.